canvas_modules/common-canvas/src/common-properties/properties-main/properties-main.jsx

```diff
@@ -350,20 +354,24 @@ class PropertiesMain extends React.Component {
 				this.setState({
 					editorSize: Size.MEDIUM
 				});
+				this.propertiesController.setCurrentEditorSize(Size.MEDIUM);
```

Comment:
I would move the setting of the editor size to a method and do the state and controller change together.